src/handler.ts

```diff
@@ -27,35 +27,36 @@ export async function handlePullRequest (context: Context): Promise<void> {
     return
   }
 
-  if(config.useReviewGroups && !config.reviewGroups){
+  if (config.useReviewGroups && !config.reviewGroups) {
     throw new Error('Error in configuration file to do with using review groups. Expected \'reviewGroups\' variable to be set because the variable \'useReviewGroups\' = true.')
   }
 
-  if(config.useAssigneeGroups && !config.assigneeGroups){
+  if (config.useAssigneeGroups && !config.assigneeGroups) {
     throw new Error('Error in configuration file to do with using review groups. Expected \'assigneeGroups\' variable to be set because the variable \'useAssigneeGroups\' = true.')
   }
 
-  const reviewers: string[] = await chooseReviewers(context, config, [], context.payload.repository.owner.login)
-  await chooseAssignees(context, config, reviewers, context.payload.repository.owner.login)
+  const reviewers: string[] = await chooseReviewers(context, config, [], context.payload.pull_request.user.login)
```

Comment:
This chenge by bug fix https://github.com/kentaro-m/auto-assign/pull/72
Fixed an issue where it was not possible to add reviewers when the repository owner created a pull request.